R/chat.R

```diff
@@ -4,8 +4,9 @@
 # trimming of the message history to fit within the context window; these
 # are left for the caller to handle in the R version.
 
-#' @importFrom htmltools tag css
+#' @importFrom htmltools tag css HTML
 #' @importFrom coro async
+#' @importFrom rlang %||%
```

Comment:
Very minor nit, but I'd prefer to let `usethis::use_import_from()` manage these. It does the same thing but stores these in a central, standard location.